images/techdocs/README.md

```diff
@@ -5,14 +5,44 @@ in Inventory.
 
 ## Usage
 
-### Running a development server
+### Docker compose configuration
+
```

Comment:
How exactly would I proceed to get a local preview site up and running?
I assume I would create a `docker-compose.yaml` somewhere in the `backstage` repo, then run `docker compose up techdocs`?

Reply:
You would need a `docker-compose.yaml` file in the repo containing the docs. I can make that more clear in the instructions.

Reply:
Ah, of course, not in backstage, but the doc repo, my bad, but clarification is great :)

Reply:
I had the same question :)
Looks very good to me!